Woche4/volume/volume.c

```diff
@@ -0,0 +1,40 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+const int HEADER_SIZE = 44;
+
+int main(int argc, char *argv[]) {
+
+    if (argc != 4) {
+        printf("Usage: ./volume input.wav output.wav factor\n");
+        return 1;
```

Comment:
```suggestion
int main(int argc, char *argv[])
{

    if (argc != 4)
    {
        printf("Usage: ./volume input.wav output.wav factor\n");
        return 1;
```

same here. be consistent with the curly brackets